doc/chapters/integration-guidelines/10-systemd-service.rst

```diff
@@ -0,0 +1,22 @@
+systemd.service
+===============
+
+As it is mentioned in :ref:`Getting Started <getting-started>` section the user API is implemented by
+the SoftwareContainer Agent as a D-bus service. The API is used to start, configure, and control
+containers. Thus in order to use SoftwareContainers the agent should be run first. To be able to start
+the agent and watch its process/system logs it would be wise decision to use systemd.service.
```

Comment:
Well, the manpage is systemd.service because it applies to any systemd service. The man page consistently says just "service" when talking about a systemd service.